An exception thrown while attempting to run tests using proxy

**Describe the bug**
Site cannot be opened while tests in running if proxy meta tag is used

**To Reproduce**
1. Run test that attempts to open some browser page with meta tag proxy, for example 
```
Meta:
@proxy

Scenario: variables initialization
Given I am on a page with the URL 'https://github.com/vividus-framework/vividus'
```

**Expected behavior**
Tests run successfully with meta tag proxy

**Artifacts**
Log with 2 different netty versions: 
```
2021-06-02 11:05:16,727 [LittleProxy-0-ProxyToServerWorker-1] ERROR org.littleshoot.proxy.impl.ProxyToServerConnection - (HANDSHAKING) [id: 0x1da8c963, L:/10.195.92.165:54132 - R:accounts.google.com/142.250.186.141:443]: Caught an exception on ProxyToServerConnection
io.netty.channel.ChannelPipelineException: io.netty.handler.ssl.SslHandler.handlerAdded() has thrown an exception; removed.
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:624) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline.addFirst(DefaultChannelPipeline.java:181) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline.addFirst(DefaultChannelPipeline.java:152) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at org.littleshoot.proxy.impl.ProxyConnection.encrypt(ProxyConnection.java:390) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ProxyConnection.encrypt(ProxyConnection.java:363) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ProxyConnection$2.execute(ProxyConnection.java:416) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ConnectionFlow.doProcessCurrentStep(ConnectionFlow.java:152) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ConnectionFlow.processCurrentStep(ConnectionFlow.java:140) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ConnectionFlow.advance(ConnectionFlow.java:107) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ConnectionFlowStep.onSuccess(ConnectionFlowStep.java:83) ~[littleproxy-2.0.0-beta-5.jar:?]
	at org.littleshoot.proxy.impl.ConnectionFlow.lambda$doProcessCurrentStep$1(ConnectionFlow.java:157) ~[littleproxy-2.0.0-beta-5.jar:?]
	at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:577) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListeners0(DefaultPromise.java:570) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:549) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:490) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:615) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:604) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:104) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.fulfillConnectPromise(AbstractNioChannel.java:300) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:335) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:702) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493) [netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.50.Final.jar:4.1.50.Final]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.lang.NoSuchFieldError: TCP_FASTOPEN_CONNECT
	at io.netty.handler.ssl.SslHandler.handlerAdded(SslHandler.java:1929) ~[netty-all-4.1.63.Final.jar:4.1.63.Final]
	at io.netty.channel.AbstractChannelHandlerContext.callHandlerAdded(AbstractChannelHandlerContext.java:938) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	at io.netty.channel.DefaultChannelPipeline.callHandlerAdded0(DefaultChannelPipeline.java:609) ~[netty-transport-4.1.50.Final.jar:4.1.50.Final]
	... 27 more

```

**Execution environment (please complete the following information):**
 - Vividus version: 0.3.6-SNAPSHOT

**Desktop Web (if applicable, please complete the following information):**
 - OS: Windows 10
 - Browser chrome

**Additional context**
Due to unknown reason there is 2 netty versions 4.1.50 and 4.1.63

--- a/vividus-plugin-web-app/build.gradle
+++ b/vividus-plugin-web-app/build.gradle
@@ -2,6 +2,7 @@ project.description = 'Vividus plugin for testing web applications'
 
 ext {
     seleniumVersion = '4.0.0-alpha-1'
+    nettyVersion = '4.1.65.Final'
 }
 
 dependencies {
@@ -26,11 +27,15 @@ dependencies {
     implementation(group: 'org.seleniumhq.selenium', name: 'selenium-opera-driver', version: seleniumVersion)
     implementation(group: 'junit', name: 'junit', version: versions.junit4)
     implementation(group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '4.4.3')
-    implementation(group: 'io.netty', name: 'netty-all', version: '4.1.65.Final')
     implementation(group: 'org.springframework', name: 'spring-context', version: versions.spring)
     implementation(group: 'org.apache.commons', name: 'commons-text', version: versions.commonsText)
     implementation(group: 'org.slf4j', name: 'slf4j-api', version: versions.slf4j)
     implementation(group: 'javax.inject', name: 'javax.inject', version: versions.javaxInject)
+
+    // browserup-proxy depends on both netty-all and netty-core, so we need to align versions of all netty dependencies
+    // https://github.com/browserup/browserup-proxy/blob/26714e26fae728733a13a0f332bd0c6dcaa78de5/browserup-proxy-core/build.gradle#L60-L76
+    implementation(group: 'io.netty', name: 'netty-all', version: "${nettyVersion}")
+    implementation(group: 'io.netty', name: 'netty-codec', version: "${nettyVersion}")
 
     compileOnly(group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: spotbugsVersion)
 

